README.md

```diff
@@ -137,6 +137,8 @@ Attestations are saved in the JSON-serialized [Sigstore bundle][6] format.
 
 If multiple subjects are being attested at the same time, a single attestation
 will be created with references to each of the supplied subjects.
+If `single-subject-attestations` is true, each attestation will be written
```

Comment:
Let's clarify this a little bit.

"If the `single-subject-attestations` option has been set to true, one attestation will be generated per provided subject and all created attestations will be written to the output file following the [JSON Lines][7] format."

Reply:
Done.